test/configuration.js

```diff
@@ -167,6 +167,33 @@ describe('configuration', function () {
       chai.config.showDiff = !chai.config.showDiff;
       assert.equal(chai.Assertion.showDiff, chai.config.showDiff);
     });
-    
+  });
+
+  describe('useProxy', function() {
+    var readNoExistentProperty = function() {
+      expect(false).to.be.tue; // typo: tue should be true
+    };
+
+    it('should have default value equal to true', function() {
+      expect(chai.config.useProxy).to.be.true;
+    });
+
+    describe('when true', function() {
+      it('should use proxy unless user\'s environment doesn\'t support', function() {
+        if (typeof Proxy !== 'undefined' || typeof Reflect !== 'undefined') {
```

Comment:
Yeah, nice catch.
Also, @meeber and @keithamus what do you think about what we talked regarding always testing the code inside the `else` clause? I'd love to hear your opinion on this matter :smile: